webdriver/tests/bidi/browsing_context/__init__.py

```diff
@@ -14,7 +14,7 @@ def assert_browsing_context(
     info,
     context,
     children=None,
-    is_root=True,
+    parent_expected=True,
```

Comment:
In the spec proposal we have `include parent id` so we should use `include_parent_id` here. I think this is more descriptive as `parent_expected`.

Reply:
I am not sure I agree: the param is the configuration for the assertion, whether it needs to expect the provided parent in the context info and I am not sure it has to match the spec because the spec does not define an assertion.